src/main/webapp/dataUseLetter/DataUseLetter.js

```diff
@@ -553,9 +557,73 @@ class DataUseLetter extends Component {
   isEmpty(value) {
     return value === '' || value === null || value === undefined;
   }
+
+  createRestriction() {
+    let restriction = this.getRestriction();
+    DataUse.createRestriction(this.props.serverUrl, restriction);
+  }
+
+  getDiseases(diseases) {
+    let diseasesResult = [];
+    diseases.map(disease => {
+      diseasesResult.push(
+        disease.key
+      );
+     });
+    return diseasesResult
```

Comment:
Pretty sure you don't need the interim `diseasesResult` variable ... the return value of `map` is an array populted with the results of whatever happened inside of the map.